Access to the raw APIs

The README file mentions that [it's possible to get access to parsing and serialization APIs](https://github.com/jsdom/whatwg-mimetype/tree/main?tab=readme-ov-file#raw-parsingserialization-apis) via `whatwg-mimetype/parser` and `whatwg-mimetype/serialize`. However, the package itself doesn't expose such modules as entry points, nor are they available on npm. Evaluating `require("whatwg-mimetype/parser")` in Node throws an error: `Cannot find module 'whatwg-mimetype/parser'`.

A workaround is to use a full path to the file:

```js
const parse = require('whatwg-mimetype/lib/parser.js');
```

I'm not sure if that's the intended use, though. The [type declarations](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5d7e8c0570c4c64da0133593c0e83edbce4a1b98/types/whatwg-mimetype/index.d.ts) maintained in the `@types` repository don't expose modules from `/lib` as well.

Is the documentation wrong? If not, could the raw APIs be exposed in the main entry point, or alternatively could more entry points be defined via [`exports`](https://nodejs.org/docs/v18.0.0/api/packages.html#package-entry-points) given that the package itself requires Node 18?

--- a/README.md
+++ b/README.md
@@ -92,8 +92,8 @@ mimeType.parameters.set("@", "x");
 If you want primitives on which to build your own API, you can get direct access to the parsing and serialization algorithms as follows:
 
 ```js
-const parse = require("whatwg-mimetype/parser");
-const serialize = require("whatwg-mimetype/serialize");
+const parse = require("whatwg-mimetype/lib/parser");
+const serialize = require("whatwg-mimetype/lib/serializer");
 ```
 
 `parse(string)` returns an object containing the `type` and `subtype` strings, plus `parameters`, which is a `Map`. This is roughly our equivalent of the spec's [MIME type record](https://mimesniff.spec.whatwg.org/#mime-type). If parsing fails, it instead returns `null`.

